src/Test/Utilities/Portable/Pdb/PdbValidation.cs

```diff
@@ -220,29 +207,55 @@ private static void RemoveEmptyMethods(XElement pdb)
-                                            e.Name == "importsforward" ||
-                                            e.Name == "xmlnamespace" ||
-                                            e.Name == "alias" ||
-                                            e.Name == "namespace" ||
-                                            e.Name == "type" ||
-                                            e.Name == "defunct" ||
-                                            e.Name == "extern" ||
-                                            e.Name == "externinfo" ||
-                                            e.Name == "local" && e.Attributes().Any(a => a.Name.LocalName == "name" && a.Value.StartsWith("$VB$ResumableLocal_"))
-                                      select e;
-
-            foreach (var e in nonPortableElements.ToArray())
+            var elementsToRemove = from e in expectedNativePdb.DescendantsAndSelf()
+                                   where e.Name == "forwardIterator" ||
+                                         e.Name == "forwardToModule" ||
+                                         e.Name == "forward" ||
+                                         e.Name == "tupleElementNames" ||
+                                         e.Name == "dynamicLocals" ||
+                                         e.Name == "using" ||
+                                         e.Name == "currentnamespace" ||
+                                         e.Name == "defaultnamespace" ||
+                                         e.Name == "importsforward" ||
+                                         e.Name == "xmlnamespace" ||
+                                         e.Name == "alias" ||
+                                         e.Name == "namespace" ||
+                                         e.Name == "type" ||
+                                         e.Name == "defunct" ||
+                                         e.Name == "extern" ||
+                                         e.Name == "externinfo" ||
+                                         e.Name == "local" && e.Attributes().Any(a => a.Name.LocalName == "name" && a.Value.StartsWith("$VB$ResumableLocal_")) ||
+                                         e.Attributes().Any(a => a.Name.LocalName == "format" && a.Value == "windows")
+                                   select e;
+
+            foreach (var e in elementsToRemove.ToArray())
             {
                 e.Remove();
             }
         }
 
+        private static void RemovePortableSpecificElements(XElement expectedNativePdb)
+        {
+            var elementsToRemove = from e in expectedNativePdb.DescendantsAndSelf()
+                                   where e.Attributes().Any(a => a.Name.LocalName == "format" && a.Value == "portable")
+                                   select e;
+
+            foreach (var e in elementsToRemove.ToArray())
+            {
+                e.Remove();
+            }
```

Comment:
The same `foreach (var e in elementsToRemove.ToArray()) { ... }` appears several times. Consider extracting a `RemoveElements(IEnumerable<XElement>)` helper method.